models.py

```diff
@@ -37,35 +37,39 @@ def make_uuid():
         unique=True,
         help_text="unique id, generated on email submission",
     )
+
     first_name = models.CharField(
         max_length=36,
     )
+
     last_name = models.CharField(
         max_length=36,
     )
-    username = models.CharField(
-        max_length=36,
-    )
+
+    username = models.CharField(max_length=36)
+
     email = models.EmailField(
         max_length=41,
         help_text="the potential member's email address",
     )
-    custom_msg = models.TextField(
-        blank=True,
-    )
+
+    custom_msg = models.TextField(blank=True)
+
     date_invited = models.DateField(
         auto_now=True,
         help_text="the day on which the superuser invited the potential member",
     )
+
     permissions = models.ManyToManyField(Permission)
     groups = models.ManyToManyField(Group)
     is_super_user = models.BooleanField()
 
     class Meta:
+        abstract = True
         ordering = ["date_invited"]
 
-    def __unicode__(self):
-        return "%s, %s: %s" % (self.last_name, self.first_name, self.date_invited)
```

Comment:
When I applied the patch I had to manually resolve a merge conflict. That method must have have mistakenly removed during the process.